run-tests.sh

```diff
@@ -25,7 +25,7 @@
 
 
 pydocstyle invenio_accounts_rest tests && \
-isort -rc -c -df **/*.py && \
+isort -m 5 -rc -c -df **/*.py && \
```

Comment:
Ah, here it is. You shouldn't change the import style. This is already defined in ``.editorconfig``  (see https://github.com/inveniosoftware/cookiecutter-invenio-module/blob/master/%7B%7B%20cookiecutter.project_shortname%20%7D%7D/.editorconfig#L16)